src/config.js

```diff
@@ -4,6 +4,65 @@ import testConfig from './config.test';
 
 const rpcHost = require('./env.json').rpcHost;
 
+const gnosis = {
+  tokenContract: '0x6810e776880C02933D47DB1b9fc05908e5386b96',
+  information: {
+    aliasName: 'Gnosis',
+    website: 'https://gnosis.pm/',
+    logo: 'https://gnosis.pm/favicon.ico?v=1',
+  },
+  events: {
+    BidSubmission: {
+      args: {
+        tokens: 'amount',
+        sender: 'sender',
+      },
+      customArgs: {
+        _from: '0x0000000000000000000000000000000000000000',
+      },
+      firstTransactionBlockNumber: 3557583,
+      lastTransactionBlockNumber: 3635851, // use block number to skip tokens created in finalize()
+      countTransactions: true,
+    },
+  },
+  icoParameters: {
+    cap: async (web3, icoContract) => {
+      const maxCap = await toPromise(icoContract.MAX_TOKENS_SOLD)().valueOf();
```

Comment:
This is good enough! But as a special task in solidity learning: there is a place where they derive ETH cap from TOKEN cap... port it to javascript or call this code directly